packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -19,22 +19,22 @@
           {{ debug.commitInfo.summary }}
         </li>
         <div
-          v-if="props.runNumber && props.commitsAhead"
-          class="border rounded border-gray-100 items-center flex text-sm h-6"
+          class="border rounded flex border-gray-100 h-6 text-sm items-center"
           data-cy="debug-runCommit-info"
         >
           <span
-            class="font-medium text-gray-700 px-2 mx-px items-center"
+            class="font-medium mx-px px-2 text-gray-700 items-center"
             data-cy="debug-runNumber"
           >
-            Run #{{ props.runNumber }}
+            Run #{{ debug.runNumber }}
           </span>
-          <div class="w-px h-3 my-6px bg-gray-100" />
+          <div class="bg-gray-100 h-3 my-6px w-px" />
           <span
-            class="font-normal text-orange-500 px-2 mx-px items-center"
+            v-if="props.commitsAhead"
+            class="font-normal mx-px px-2 text-orange-500 items-center"
             data-cy="debug-commitsAhead"
           >
-            {{ props.commitsAhead }}
+            {{ t('debugPage.header.commitsAhead', {commits: props.commitsAhead}) }}
```

Comment:
`vue-i18n` has built-in support for auto-pluralization that would be ideal here so we don't display a message like "You are 1 commits ahead". See the `runner.selectorPlayground.matches` in `en-US.json` for an example